gnovm/docs/go-gno-compatibility.md

```diff
@@ -6,33 +6,59 @@
 
 Legend: full, partial, missing, TBD.
 
-| keyword     | status |
-|-------------|--------|
-| break       | TBD    |
-| case        | TBD    |
-| chan        | TBD    |
-| const       | TBD    |
-| continue    | TBD    |
-| default     | TBD    |
-| defer       | TBD    |
-| else        | TBD    |
-| fallthrough | TBD    |
-| for         | TBD    |
-| func        | TBD    |
-| go          | TBD    |
-| goto        | TBD    |
-| if          | TBD    |
-| import      | TBD    |
-| interface   | TBD    |
-| map         | TBD    |
-| package     | TBD    |
-| range       | TBD    |
-| return      | TBD    |
-| select      | TBD    |
-| struct      | TBD    |
-| switch      | TBD    |
-| type        | TBD    |
-| var         | TBD    |
+| keyword     | support                |
+|-------------|------------------------|
+| break       | full                   |
+| case        | full                   |
+| const       | full                   |
+| continue    | full                   |
+| default     | full                   |
+| defer       | full                   |
+| else        | full                   |
+| fallthrough | full                   |
+| for         | full                   |
+| func        | full                   |
+| go          | missing (after launch) |
+| goto        | full                   |
+| if          | full                   |
+| import      | full                   |
+| interface   | full                   |
+| package     | full                   |
+| range       | full                   |
+| return      | full                   |
+| select      | missing (after launch) |
+| struct      | full                   |
+| switch      | full                   |
+| type        | full                   |
+| var         | full                   |
+
+## Native types
+
+| type                                          | usage                  | persistency                                                |
+|-----------------------------------------------|------------------------|------------------------------------------------------------|
+| `bool`                                        | full                   | full                                                       |
+| `byte`                                        | full                   | full                                                       |
+| `float32`, `float64`                          | full                   | full (determinism to confirm across various architectures) |
```

Comment:
Isn't this just IEEE-754? The behaviour is deterministic and the same on all modern processors; it's just that as always, floating point arithmetic gets creative sometimes.

Reply:
Probably, let's open an issue now or wait for someone to tackle this :)

Reply:
Probably, let's open an issue now or wait for someone to tackle this :)

Reply:
To me it's a "deterministic until proven otherwise, in which case it's a bug", so there's no need for it. But approving; if you see what you said as a better fit then let's merge.

Reply:
I agree, let's say this instead, thanks.